compiler/noirc_evaluator/src/ssa/ir/printer.rs

```diff
@@ -264,8 +264,10 @@ fn display_instruction_inner(
         Instruction::DecrementRc { value } => {
             writeln!(f, "dec_rc {}", show(*value))
         }
-        Instruction::RangeCheck { value, max_bit_size, .. } => {
-            writeln!(f, "range_check {} to {} bits", show(*value), *max_bit_size,)
+        Instruction::RangeCheck { value, max_bit_size, assert_message } => {
+            let message =
+                assert_message.as_ref().map(|message| format!(", {message:?}")).unwrap_or_default();
```

Comment:
```suggestion
                assert_message.as_ref().map(|message| format!(", {message}")).unwrap_or_default();
```
Doesn't seem to make any difference, but it confused me.